src/univalue/CMakeLists.txt

```
@@ -15,11 +15,120 @@ target_include_directories(univalue
+  generate_header_from_json(test/round6.json)
+  generate_header_from_json(test/round7.json)
+  add_executable(unitester
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail10.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail11.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail12.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail13.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail14.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail15.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail16.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail17.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail18.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail19.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail1.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail20.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail21.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail22.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail23.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail24.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail25.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail26.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail27.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail28.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail29.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail2.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail30.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail31.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail32.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail33.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail34.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail35.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail36.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail37.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail38.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail39.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail3.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail40.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail41.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail42.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail44.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail45.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail4.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail5.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail6.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail7.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail8.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/fail9.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/pass1.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/pass2.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/pass3.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/pass4.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round1.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round2.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round3.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round4.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round5.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round6.json.h
+    ${CMAKE_CURRENT_BINARY_DIR}/test/round7.json.h
+    test/unitester.cpp)
```

style nit: Follow surrounding style:
```suggestion
    test/unitester.cpp
  )
```
?